core/shared/src/main/scala/zio/prelude/ZValidation.scala

```diff
@@ -163,6 +163,15 @@ sealed trait ZValidation[+W, +E, +A] { self =>
       case Success(w, a) => Success(f(w), a)
     }
 
+  final def or[E1, W1 >: W, A1 >: A](that: ZValidation[W1, E1, A1])(implicit ev: E <:< W1): ZValidation[W1, E1, A1] =
```

Comment:
This is still embedding the idea of moving the errors from the first effect into the log channel which is perfectly reasonable but also not necessarily what you want to do, especially since it requires going from a `Validation` to a `ZValidation` which you may not be using. I would suggest renaming `or` to `orElseLog` and `orWith` to `orElseLogWith`.

```scala
final def orElse[W1

Reply:
👍 I've also been thinking about what other combinators would be reasonable.  I do like the idea of not specifically requiring the log gets moved over.